reporter-web-app/src/App.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Copyright (C) 2017-2021 HERE Europe B.V.
```

Comment:
Commit message: Could you maybe add a link to https://ant.design/components/icon/ so people get an idea what these icons look like beforehand?